src/main/java/calculator/StringAddCalculator.java

```diff
@@ -0,0 +1,67 @@
+package calculator;
+
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class StringAddCalculator {
+    private static final Pattern CUSTOM_DELIMITER_PATTERN = Pattern.compile("^//(.)\\n(.*)$");
+    private static final String COMMA_OR_COLON_DELIMITER = "[,:]";
+    private static final int NULL_AND_BLANK_VALUE = 0;
+    private static final int MATCHER_CUSTOM_DELIMITER = 1;
+    private static final int MATCHER_TEXT = 2;
+
+    public static int splitAndSum(String text) {
+        if(checkNullAndBlank(text)) {
+            return NULL_AND_BLANK_VALUE;
+        }
+
+        Matcher matcher = CUSTOM_DELIMITER_PATTERN.matcher(text);
+        if(matcher.find()) {
+            return sum(toIntegerList(splitDelimiter(
+                    matcher.group(MATCHER_CUSTOM_DELIMITER),matcher.group(MATCHER_TEXT))));
+        }
+
+        return sum(toIntegerList(splitDelimiter(null,text)));
```

Comment:
```suggestion
        return sum(toIntegerList(splitDelimiter(COMMA_OR_COLON_DELIMITER, text)));
```
이렇게 써보면 어떨까요?

splitDelimiter 메서드도 깔끔해지고 정말 자르기만 할 수 있게 만들수 있을 것 같아요!

Reply:
맞습니다! 제가 놓친 부분이였습니다!